jme3-bullet/src/common/java/com/jme3/bullet/control/RigidBodyControl.java

```diff
@@ -266,16 +269,20 @@ public void update(float tpf) {
     public void render(RenderManager rm, ViewPort vp) {
     }
 
-    public void setPhysicsSpace(PhysicsSpace space) {
+    public void setPhysicsSpace(final PhysicsSpace space) {
+        final PhysicsSpace currentSpace = getPhysicsSpace();
```

Comment:
Thinking on it some more.  I don't like and it needs to be removed.  setPhysicsSpace() is managing the state of this.space and shouldn't have to consult its own getter to know that.  To any reasonable Java developer, this.space is unambiguously "the current space"... and certainly the most current "current space" that it's ever possible for setPhysicsSpace() (the one managing that field) to deal with.  Calling getPhysicsSpace() is extra indirection that may obscure some random bit of horror that makes this method fail... as a developer, I'd have to go look.  And that's pointless since we are now (at best) indirectly asking about the one field that this method is supposed to be managing itself.

In general, your parches are more likely to be accepted if they only include the things you actually want to change and not a bunch of random other 'style' stuff.

tl;dr: some peoples "more readable" is another's unnecessary obfuscation.